UI/frontend-plugins/decklink-output-ui/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+project(decklink-output-ui)
+
+if (APPLE)
+	find_library(COCOA Cocoa)
+	include_directories(${COCOA})
+endif ()
```

Comment:
Style:  It seems like the preferred style in CMake is `endif()`.  Remove the space after `endif`.